nservicebus/security/encryption.md

```diff
@@ -96,12 +108,19 @@ This allows you to explicitly handled the encryption and decryption of each valu
 
 ## Message Encryption
 
-Message encryption leverages the pipeline to apply encryption to the whole message body.
+Message encryption encrypts the complete message body. It can even include the message headers if these would contain sensitive data.
 
-Once way of achieving this is using a `IMutateTransportMessages`.
+We currently do not offer this feature but it can be achieved by creating a transport message mutator using `IMutateTransportMessages`
 
 <!-- import MessageBodyEncryptor -->
 
-The this class can be then injected into the container using the following
+This class can be then injected into the container using the following:
 
 <!-- import UsingMessageBodyEncryptor -->
+
+
+## Transport Encryption
+
+However, using message encryption is often used to prevent intercepting data during transport. If this is the requirement then you can maybe on the transport for encryption. RabbitMQ, SQL Server, Azure Storage Queues and Azure Service Bus support SSL.
```

Comment:
Agree with the however.

Not entirely true, message encryption makes a lot of sense if you rely on message infrastructure for communication. If the data is encrypted than it does not have a performance impact if it has multiple hops and maybe is even stored. That is why I explicitly mention 'if its used to prevent intercepting data during transport'.

 I hope you understand what I'm trying to say. I think I lack skills to summarize this correctly.